client/my-sites/plans-features-main/header.jsx

```diff
@@ -0,0 +1,25 @@
+/** @format */
```

Comment:
The prettier pragmas are no longer necessary AFAIK.

Reply:
Good to know, thanks! I'll remove it in the next PR.